src/patternfly/utilities/variables.scss

```diff
@@ -32,7 +32,7 @@ $pf-global--dark-off--BackgroundColor:   $pf-color-black-800 !default;
 
 // Text color
 $pf-global--dark-color:       $pf-color-black-900 !default;
-$pf-global-dark-off-color:    $pf-color-black-400 !default;
+$pf-global-dark-off-color:    $pf-color-black-600 !default;
```

Comment:
btw, I made this change after an in-person review with @jennyhaines

Reply:
Yes- the reasoning is because pf-black-600 is the lightest accessible color for our light backgrounds.